src/modules/account/account.repository.ts

```diff
@@ -81,6 +80,37 @@ export class AccountRepository {
     return response;
   }
 
+  async getCredentialId(id: string): Promise<GetCredential | void> {
+    const response = await this.prisma.credential
+      .findFirst({
```

Comment:
O método `findFirst` passa por todos os dados da tabela para buscar o primeiro que aparece, já o `findUnique` usa o índice criado por ser um dado único.

```suggestion
      .findUnique({
```